src/todos/todos.service.ts

```diff
@@ -22,4 +22,11 @@ export class TodosService {
     }
     return foundTodo;
   }
+
+  async updateTodoById(id: string, updateTodo: UpdateTodoDto): Promise<Todo> {
```

Comment:
```suggestion
  async updateOne(id: string, updateTodo: UpdateTodoDto): Promise<Todo> {
```

No need to specify `todo` as this is in todosService.